src/drizzleReactHooks.js

```diff
@@ -0,0 +1,203 @@
+import React, {
```

Comment:
This file is kind of long (at 200+ lines), do you think you can break this up and maybe put it inside a `src/hooks` folder?

Reply:
Sure